File: examples/threat_level.js
// Node.js / Deno / Bun
const resp = await fetch("https://estwarden.eu/api/threat-index");
const { level, score, date } = await resp.json();
const emoji = { GREEN: "🟢", YELLOW: "🟡", ORANGE: "🟠", RED: "🔴" }[level] || "⚪";
console.log(`${emoji} ${level} — CTI: ${score.toFixed(1)}/100 (${date})`);

File: examples/threat_level.py
#!/usr/bin/env python3
"""Get current Baltic threat level."""
import json, urllib.request

with urllib.request.urlopen("https://estwarden.eu/api/threat-index") as r:
    d = json.loads(r.read())

emoji = {"GREEN": "🟢", "YELLOW": "🟡", "ORANGE": "🟠", "RED": "🔴"}.get(d["level"], "⚪")
print(f"{emoji} {d['level']} — CTI: {d['score']:.1f}/100 ({d['date']})")
